public/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="./css/style.css">
+    <script src="./js/jquery-3.5.1.js"></script>
+    <title>Bowling</title>
+  </head>
+  <body>
+    <section>
+      <h1 id="score"></h1>
+      <fieldset>
+        <label for ="bowl_input">How many pins did you knock over?</label>
```

Comment:
Nitpit: Naming conventions are annoying different for every language. I looked up for html/css this should be bowl-input

Reply:
Ah thanks!